website/content/middleware/jwt.md

```diff
@@ -27,50 +27,92 @@ e.Use(middleware.JWTWithConfig(middleware.JWTConfig{
 }))
 ```
 
+*Usage with different JWT library/repository*
+
+> NB: Due backwards compatibility reasons Echo still uses unmaintained, with potential security risks `dgrijalva/jwt-go` version. We recommend using maintained fork ala https://github.com/golang-jwt/jwt
+
+```go
+// Use maintained fork (https://github.com/golang-jwt/jwt) of `dgrijalva/jwt-go` instead
```

Comment:
Use maintained fork (https://github.com/golang-jwt/jwt) instead of `dgrijalva/jwt-go`